<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="21" failures="0" skipped="0" tests="21" time="64.433" timestamp="2024-08-10T12:49:53.330024" hostname="DESKTOP-BAHKCM1"><testcase classname="" name="Basic_python.Basic program.kiran feature branch.test_login" time="0.000"><error message="collection failure">Basic_python\Basic program\kiran feature branch\test_login.py:9: in &lt;module&gt;
    driver.find_element(By.XPATH, "//input[@id='user-input']").send_keys("0000020232")
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\webdriver.py:741: in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\webdriver.py:347: in execute
    self.error_handler.check_response(response)
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
E   from unknown error: web view not found
E     (Session info: chrome=127.0.6533.100)
E   Stacktrace:
E   	GetHandleVerifier [0x00007FF6D74D9632+30946]
E   	(No symbol) [0x00007FF6D748E3C9]
E   	(No symbol) [0x00007FF6D7386FDA]
E   	(No symbol) [0x00007FF6D735CB85]
E   	(No symbol) [0x00007FF6D74037A7]
E   	(No symbol) [0x00007FF6D741A771]
E   	(No symbol) [0x00007FF6D73FC813]
E   	(No symbol) [0x00007FF6D73CA6E5]
E   	(No symbol) [0x00007FF6D73CB021]
E   	GetHandleVerifier [0x00007FF6D760F83D+1301229]
E   	GetHandleVerifier [0x00007FF6D761BDB7+1351783]
E   	GetHandleVerifier [0x00007FF6D7612A03+1313971]
E   	GetHandleVerifier [0x00007FF6D750DD06+245686]
E   	(No symbol) [0x00007FF6D749758F]
E   	(No symbol) [0x00007FF6D7493804]
E   	(No symbol) [0x00007FF6D7493992]
E   	(No symbol) [0x00007FF6D748A3EF]
E   	BaseThreadInitThunk [0x00007FFA6AE87374+20]
E   	RtlUserThreadStart [0x00007FFA6B6FCC91+33]</error></testcase><testcase classname="" name="practice_assignment.SeleniumCode.AutomationFramework.tests" time="0.000"><error message="collection failure">C:\python_3.12\Lib\importlib\__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1387: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1360: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1331: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:935: in _load_unlocked
    ???
C:\python_3.12\Lib\site-packages\_pytest\assertion\rewrite.py:178: in exec_module
    exec(co, module.__dict__)
practice_assignment\SeleniumCode\AutomationFramework\tests\conftest.py:2: in &lt;module&gt;
    from base.WebdriverFactory import WebdriverFactory
E   ModuleNotFoundError: No module named 'base'</error></testcase><testcase classname="" name="practice_assignment.SeleniumCode.PytestFramework.test.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\Mini_Framework_POM\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\PytestFramework\test\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="practice_assignment.SeleniumCode.PytestFramework.test.test_html_report" time="0.000"><error message="collection failure">practice_assignment\SeleniumCode\PytestFramework\test\test_html_report.py:20: in &lt;module&gt;
    @pytest.mark.parametrize("greeting", greetings_msg)
E   NameError: name 'greetings_msg' is not defined</error></testcase><testcase classname="" name="practice_assignment.SeleniumCode.PytestFramework.test.test_parametrize_values" time="0.000"><error message="collection failure">practice_assignment\SeleniumCode\PytestFramework\test\test_parametrize_values.py:16: in &lt;module&gt;
    @pytest.mark.parametrize("greeting", greetings_msg)
E   NameError: name 'greetings_msg' is not defined</error></testcase><testcase classname="" name="python_selenium.Automation_framework.test" time="0.000"><error message="collection failure">C:\python_3.12\Lib\importlib\__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1387: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1360: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1331: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:935: in _load_unlocked
    ???
C:\python_3.12\Lib\site-packages\_pytest\assertion\rewrite.py:178: in exec_module
    exec(co, module.__dict__)
python_selenium\Automation_framework\test\conftest.py:3: in &lt;module&gt;
    from data.dummy_booking_data.dummy_test_data import *
E   ModuleNotFoundError: No module named 'data.dummy_booking_data'</error></testcase><testcase classname="" name="python_selenium.mini_framework.Framework_page_object_module_POM.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\Mini_Framework_POM\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\mini_framework\Framework_page_object_module_POM\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="python_selenium.mini_framework.Framework_page_object_module_POM.test_feature_class" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_feature_class' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\Mini_Framework_POM\test_feature_class.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\mini_framework\Framework_page_object_module_POM\test_feature_class.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="python_selenium.selenium.inifile_withtest.test_selenium_code_class" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_selenium_code_class' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\PytestFramework\test\test_selenium_code_class.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\selenium\inifile_withtest\test_selenium_code_class.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="python_selenium.selenium.inifile_withtest.test_smoke_class" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_smoke_class' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\PytestFramework\test\test_smoke_class.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\selenium\inifile_withtest\test_smoke_class.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="python_selenium.selenium.test_folder.test_file1with_selenium" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_file1with_selenium' has this __file__ attribute:
  E:\Python_Selenium\project branch\python_selenium\selenium\fixture\test_file1with_selenium.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\selenium\test_folder\test_file1with_selenium.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="python_selenium.selenium.test_folder.test_fixture_with_selenium" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_fixture_with_selenium' has this __file__ attribute:
  E:\Python_Selenium\project branch\python_selenium\selenium\fixture\test_fixture_with_selenium.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\python_selenium\selenium\test_folder\test_fixture_with_selenium.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="teachmint_testing_with_selenium.test_login" time="0.000"><error message="collection failure">teachmint_testing_with_selenium\test_login.py:9: in &lt;module&gt;
    driver.find_element(By.XPATH, "//input[@id='user-input']").send_keys("0000020232")
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\webdriver.py:741: in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\webdriver.py:347: in execute
    self.error_handler.check_response(response)
C:\python_3.12\Lib\site-packages\selenium\webdriver\remote\errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
E   from unknown error: web view not found
E     (Session info: chrome=127.0.6533.100)
E   Stacktrace:
E   	GetHandleVerifier [0x00007FF6D74D9632+30946]
E   	(No symbol) [0x00007FF6D748E3C9]
E   	(No symbol) [0x00007FF6D7386FDA]
E   	(No symbol) [0x00007FF6D735CB85]
E   	(No symbol) [0x00007FF6D74037A7]
E   	(No symbol) [0x00007FF6D741A771]
E   	(No symbol) [0x00007FF6D73FC813]
E   	(No symbol) [0x00007FF6D73CA6E5]
E   	(No symbol) [0x00007FF6D73CB021]
E   	GetHandleVerifier [0x00007FF6D760F83D+1301229]
E   	GetHandleVerifier [0x00007FF6D761BDB7+1351783]
E   	GetHandleVerifier [0x00007FF6D7612A03+1313971]
E   	GetHandleVerifier [0x00007FF6D750DD06+245686]
E   	(No symbol) [0x00007FF6D749758F]
E   	(No symbol) [0x00007FF6D7493804]
E   	(No symbol) [0x00007FF6D7493992]
E   	(No symbol) [0x00007FF6D748A3EF]
E   	BaseThreadInitThunk [0x00007FFA6AE87374+20]
E   	RtlUserThreadStart [0x00007FFA6B6FCC91+33]</error></testcase><testcase classname="" name="techmint_website_testing.data.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'data.test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\project_file\data\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\techmint_website_testing\data\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="techmint_website_testing.test" time="0.000"><error message="collection failure">C:\python_3.12\Lib\site-packages\_pytest\config\__init__.py:717: in _importconftest
    mod = import_path(
C:\python_3.12\Lib\site-packages\_pytest\pathlib.py:614: in import_path
    raise ImportPathMismatchError(module_name, module_file, path)
E   _pytest.pathlib.ImportPathMismatchError: ('test.conftest', 'E:\\Python_Selenium\\project branch\\project_file\\test\\conftest.py', WindowsPath('E:/Python_Selenium/project branch/techmint_website_testing/test/conftest.py'))</error></testcase><testcase classname="" name="testing_mercury_travel_website.data.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'data.test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\project_file\data\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\testing_mercury_travel_website\data\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="testing_mercury_travel_website.test" time="0.000"><error message="collection failure">C:\python_3.12\Lib\site-packages\_pytest\config\__init__.py:717: in _importconftest
    mod = import_path(
C:\python_3.12\Lib\site-packages\_pytest\pathlib.py:614: in import_path
    raise ImportPathMismatchError(module_name, module_file, path)
E   _pytest.pathlib.ImportPathMismatchError: ('test.conftest', 'E:\\Python_Selenium\\project branch\\project_file\\test\\conftest.py', WindowsPath('E:/Python_Selenium/project branch/testing_mercury_travel_website/test/conftest.py'))</error></testcase><testcase classname="" name="ticket_booking_using_folders_with_pytest.data.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'data.test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\project_file\data\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\ticket_booking_using_folders_with_pytest\data\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="ticket_booking_using_folders_with_pytest.test" time="0.000"><error message="collection failure">C:\python_3.12\Lib\site-packages\_pytest\config\__init__.py:717: in _importconftest
    mod = import_path(
C:\python_3.12\Lib\site-packages\_pytest\pathlib.py:614: in import_path
    raise ImportPathMismatchError(module_name, module_file, path)
E   _pytest.pathlib.ImportPathMismatchError: ('test.conftest', 'E:\\Python_Selenium\\project branch\\project_file\\test\\conftest.py', WindowsPath('E:/Python_Selenium/project branch/ticket_booking_using_folders_with_pytest/test/conftest.py'))</error></testcase><testcase classname="" name="website_testing_using_pytest.test_data" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_data' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\Mini_Framework_POM\test_data.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\website_testing_using_pytest\test_data.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase><testcase classname="" name="website_testing_using_pytest.test_feature_class" time="0.000"><error message="collection failure">import file mismatch:
imported module 'test_feature_class' has this __file__ attribute:
  E:\Python_Selenium\project branch\practice_assignment\SeleniumCode\Mini_Framework_POM\test_feature_class.py
which is not the same as the test file we want to collect:
  E:\Python_Selenium\project branch\website_testing_using_pytest\test_feature_class.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</error></testcase></testsuite></testsuites>